### PR TITLE
aligning the external link + header capitalised 

### PR DESCRIPTION
**Pull request by legendarykamal**
Signed-off-by: Kamal Nayan <95926324+legendarykamal@users.noreply.github.com>

**Description**

1> The external links looked offset so -> aligned them 
2> changed the header on the home page 

Small changes but give more proffesional look.

This PR fixes #

**Notes for Reviewers**
Earlier:- 
image

Now:- 
image

image

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
The menus still overlap when one is opening and the other closing... or this is PR not addressing the issue that you picked up?

**Comment by legendarykamal** [comment]
hi @leecalcote that issue has been solved in this PR https://github.com/meshery/meshery.io/pull/830 .

while this PR tries to achieve things mentioned in the desc which in 
1> The external links looked offset so -> aligned them
2> changed the header on the home page

**Comment by Rohith-Raju** [comment]
@legendarykamal looks good other than that!!

**Comment by legendarykamal** [comment]
> @legendarykamal looks good other than that!!

thanks @Rohith-Raju lets get this merged :smiley:

File: _includes/homePage.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 <section class="hero" style="padding-bottom: 1rem">
 	<div class="text-container">
-		<h1 style="margin: 3rem 0rem 3rem 0rem;">the <span style="font-weight: 300 !important">extensible</span> Kubernetes manager</h1>
+		<h1 style="margin: 3rem 0rem 3rem 0rem;">The <span style="font-weight: 300 !important">extensible</span> Kubernetes manager</h1>
```
Comment (leecalcote) [inline _includes/homePage.html:17]: Revert, please

File: _includes/navigation.html [hunk 2]
```diff
@@ -36,7 +36,7 @@
 							class="subnav-item">{{sublink.name}}
 							{% if sublink.external %}
 							<img src="/images/buttons/external-link_gray.svg"
-							style="max-width:  10px; vertical-align:top;" />
+							style="max-width:  10px; vertical-align:center;" />
```
Comment (leecalcote) [inline _includes/navigation.html:39]: good

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: _includes/homePage.html (hunk 1 above)
Comment (legendarykamal) [reply _includes/homePage.html:17]: Oh, so that was on purpose ok ok :+1:

File: _includes/navigation.html (hunk 2 above)
Comment (legendarykamal) [reply _includes/navigation.html:39]: :smile:

**Comment by leecalcote** [comment]
Thanks for this, @legendarykamal. The superscript placement of the external link icons is intentional. Given that the both of the suggested changes in this PR are not desired, I'll close this PR.

**Pull request closed without merging**